### PR TITLE
Amend: App health checks for 400 responses

### PR DESCRIPTION
**Pull request by andygout**
cc @ironsidevsquincy 

Have done some examining of spikes of 400 errors throughout Nov and a threshold of 35 would seem to catch two of them (04 and 11 Nov; the latter being the instance we'd definitely want to catch) but not the others.

next-api-400s-nov-2016

I wonder if it's worth increasing the severity to alert when a substantial and sustained enough number of bad requests are happening. Even though it's the downstream client that will be erroring, it will mean we get a good heads-up (as would have been helpful [here](https://github.com/Financial-Times/next-bugs/blob/master/incidents/2016-11-12%20The%20one%20were%20the%20front-page%20went%20pop.md)).

### CODE REVIEW

File: server/config/health-checks/app.js [hunk 1]
```diff
@@ -9,9 +9,9 @@ export default {
 			technicalSummary: 'Check there haven‘t been more than 10 400 responses from the app in the last 10 minutes',
```
Comment (ironsidevsquincy) [inline server/config/health-checks/app.js]: will need to update description

File: server/config/health-checks/app.js [hunk 2]
```diff
@@ -9,9 +9,9 @@ export default {
...
 			panicGuide: `Check Grafana (http://grafana.ft.com/dashboard/db/next-api?panelId=36&fullscreen&var-region=${process.env.REGION || 'EU'}) for \`400\` related errors and Splunk (http://bit.ly/2glHW51)`,
 			type: 'graphiteThreshold',
-			metric: `heroku.api.web_*_${process.env.REGION || 'EU'}.express.query_*.res.status.400.count`,
+			metric: `integral(heroku.api.web_*_${process.env.REGION || 'EU'}.express.query_*.res.status.400.count)`,
 			samplePeriod: '10min',
-			threshold: 10
+			threshold: 35
```
Comment (ironsidevsquincy) [inline server/config/health-checks/app.js]: not too high?

**Comment by ironsidevsquincy** [comment]
Don't think we should change the severity of this, as it's more a heads up then a genuine error we'd want to wake people up at four in the morning with
But could do that for the other status codes (5xx responses)

File: server/config/health-checks/app.js (hunk 2 above)
Comment (andygout) [inline server/config/health-checks/app.js]: Seemed to be a middle ground so would trigger in a way that would cover the instance that we've identified as a problem, but not too often so as to become noise.

Having said that, setting the threshold as 30, would incite one more additional instance, and 25 another one on top of that (for instances this month). So maybe 25?

**Pull request merged**